src/utils/index.tsx

```diff
@@ -228,3 +229,51 @@ export function getQueryClient() {
 
     return queryClient
 }
+
+const SiSymbol = ['', 'k', 'M', 'G', 'T', 'P']
+
+/**
+ * Abbreviates BigNumberish value.
+ */
+export function abbr(
```

Comment:
@tumppi no `log10` needed. LMK if you see any issues with this approach!

Reply:
That’s clever! No math needed 👍🏻